### ISSUE TITLE
`v1.0.0` : Range specification for `iss:locationRINF`

### ISSUE THREAD

**Issue by StephaneBranly** [Interoperable-data/ISS_vocabulary#6]
Should the object property `iss:locationRINF` have its range specified?

**Comment by gatemezing** [comment]
Good point! The range could be ``[era:InfrastractureObject](https://data-interop.era.europa.eu/era-vocabulary/#InfrastructureObject)`. We'll discuss that internally.
